File: Backend/MultiSportTrainerAPI/MultiSportTrainerAPI/DTOs/AuthResponse.cs
namespace MultiSportTrainerAPI.DTOs
{
    public class AuthResponse
    {
        public int UserId { get; set; }

        public string FullName { get; set; } = string.Empty;

        public string Email { get; set; } = string.Empty;

        public string Role { get; set; } = string.Empty;

        public string? SportFocus { get; set; }

        public string Token { get; set; } = string.Empty;

        public string Message { get; set; } = string.Empty;
    }
}

File: AndroidApp/MultiSportTrainer-Mobile/app/src/main/java/com/example/multisporttrainer/models/AuthResponse.java
package com.example.multisporttrainer.models;

public class AuthResponse {

    private int userId;
    private String fullName;
    private String email;
    private String role;
    private String sportFocus;
    private String token;
    private String message;

    public int getUserId() {
        return userId;
    }

    public String getFullName() {
        return fullName;
    }

    public String getEmail() {
        return email;
    }

    public String getRole() {
        return role;
    }

    public String getSportFocus() {
        return sportFocus;
    }

    public String getToken() {
        return token;
    }

    public String getMessage() {
        return message;
    }
}